var/spack/repos/builtin/packages/sina/package.py

```diff
@@ -0,0 +1,101 @@
+import os
+
+from spack import *
+
+
+class Sina(CachedCMakePackage):
+    """Sina C++ Library"""
+
+    homepage = 'https://github.com/LLNL/Sina'
+    url = 'https://github.com/LLNL/Sina/releases/download/v1.10.0/sina-cpp-1.10.0.tar.gz'
+
+    version('develop', git='git@github.com:LLNL/Sina.git',
+            submodules=True, branch='develop')
+    version('1.10.0', sha256='b34379ce8cc5eca5a0f16893053fac75be14c2109d1beed4c6d48e11f9b281c7')
+
+    variant('docs', default=False,
+            description='Allow generating documentation')
+    variant('adiak', default=False,
+            description='Create interface for calling Sina through Adiak')
+    # Availability of the adiak version with optional mpi varies
+    variant('adiak-no-mpi', default=False,
+            description='Create interface for calling Sina through Adiak')
+    variant('test', default=False,
+            description='Build tests')
+
+    depends_on('cmake@3.8.0:', type='build')
+    depends_on('adiak', when='+adiak')
+    depends_on('mpi', when='+adiak')
+    depends_on('adiak ~mpi', when='+adiak-no-mpi')
+    depends_on('doxygen', type='build', when='+docs')
+    depends_on('conduit ~python ~mpi ~hdf5 ~silo ~shared ~test ~hdf5_compat')
```

Comment:
this means that nobody can use `sina` with conduit with `python`, etc. enabled.  These types of restrictions really belong in the environment you're building your app in, not in `sina`.